### DISCUSSION TITLE
Automatic similarity between primary key and foreign key makes the foreign key adopt the extra fluff of the primary key.

### DISCUSSION THREAD

**Discussion by MartinJohannessen** [dineug/erd-editor#352] [Q&A]
(I'm learning so I might be wrong) Concretely, when typing the primary key as a PostgreSQL's "serial" type, meaning it auto iterates, I don't think it would be correct for the foreign key to also be typed "serial" since in that instance it's a normal integer.

**Comment by dineug** [comment]
In this case, you should [disable this option and use it.](https://docs.erd-editor.io/docs/guide/guides/settings#relationship-data-type-sync)
<img width="388" alt="스크린샷 2024-02-02 234707" src="https://github.com/dineug/erd-editor/assets/45829489/fad17ca7-64cb-4ed6-9a4d-ece9ab769500">
